### ISSUE TITLE
Improved targeting for global styles, such as reset styles

### ISSUE THREAD

**Issue by boonebgorges** [PressForward/pressforward#1169]
Reset styles were introduced early in the history of PressForward. I think the goal is that they zero out WP's styles so that the Bootstrap styles work in a predictable way. But they're unscoped, so that aspects of the Dashboard outside of PF's intended area of control - such as the Dashboard background and the fonts used in the admin menu - are modified by PF. We should figure out how best to scope these styles so that they only affect PF content.

**Comment by boonebgorges** [comment]
See also #1168, #1159

**Comment by boonebgorges** [comment]
Reset styles are now loaded into a `.pf_container` scope, built using SASS directly from the Bootstrap assets loaded from NPM. These changes changed the specificity of Bootstrap's styling, which meant that the order of precedence for PF overrides was changed. In those cases, I've updated PF's selectors to ensure that our styling continues to be honored.

There may be further cleanup work to do here, but I'll close the ticket because the primary tasks are complete.
